docs/source/more_types.rst

```diff
@@ -608,12 +608,12 @@ value of type ``typing.Coroutine[Any, Any, T]``, which is a subtype of
     an expression.
 
 If you want to use coroutines in Python 3.4, which does not support
-the ``async def`` syntax, you can instead use the ``@asyncio.coroutine``
+the ``async def`` syntax, you can instead use the :py:func:`@asyncio.coroutine <asyncio.coroutine>`
 decorator to convert a generator into a coroutine.
 
 Note that we set the ``YieldType`` of the generator to be ``Any`` in the
 following example. This is because the exact yield type is an implementation
-detail of the coroutine runner (e.g. the ``asyncio`` event loop) and your
+detail of the coroutine runner (e.g. the :py:mod:`asyncio` :doc:`event loop <library/asyncio-eventloop>`) and your
```

Comment:
I think an additional link to the [event loop](https://docs.python.org/3/library/asyncio-eventloop.html) doc here would be helpful.

Reply:
I think a single link is sufficient here. The asyncio event loop docs are not directly relevant here anyway.

Reply:
Agree, removed!